### PR TITLE
Adjusted AppServiceProvider to instanciate a class depending on APP_E…

### PR DESCRIPTION
**Pull request by batare1ka**
…NV variable

### CODE REVIEW

File: app/Providers/AppServiceProvider.php [hunk 1]
```diff
@@ -16,7 +17,9 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register()
     {
-        $this->app->bind(RequestActivityLoggerInterface::class, DummyRequestActivityLogger::class);
+        $this->app->bind(RequestActivityLoggerInterface::class, function (){
+            return $this->app->make(env('APP_ENV') != 'production' ? DebugRequestActivityLogger::class : ProductionRequestActivityLogger::class);
```
Comment (egorbwork) [inline app/Providers/AppServiceProvider.php]: env helper won't work in the cached state of application so you need to use instead configs.

**Review by egorbwork — CHANGES_REQUESTED** [review]
A fix is required.

File: app/Providers/AppServiceProvider.php (hunk 1 above)
Comment (batare1ka) [reply app/Providers/AppServiceProvider.php]: changed

**Review by egorbwork — APPROVED** [review]
Now looks acceptable

**Pull request merged**